build/secondary/third_party/stb/BUILD.gn

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+source_root = "//third_party/stb"
+
+source_set("stb_truetype") {
```

Comment:
We could put a tertiary build dir tree that in the engine itself.

Reply:
This would be great. I added an issue about it here: https://github.com/flutter/flutter/issues/132981.

If we move this build config into the engine tree, we should also move the other testonly deps I've added in the past like `imgui` and `tinygltf`, as they're in more or less the same situation.